internal/storage/storage.go

```diff
@@ -83,7 +83,12 @@ func GetHoldersPage(s Storage, from, count uint64) (ret []Account, pagination *m
 		end = pagination.Total
 	}
 	pagination.End = end
-	ret = holders[from:end]
+
+	var balancesString []AccountReturn
```

Comment:
you can create an array of fixed size as you know count of holders. So it won't be resizing(recreating a larger one basically)